strawberryfields/backends/tfbackend/__init__.py

```diff
@@ -139,7 +139,6 @@
 
 """
 import sys
-from .backend import TFBackend
```

Comment:
Wait, when did this get moved up here? 🤔 

I'm looking at an older version of the file I have (locally), and it looks like this:

![image](https://user-images.githubusercontent.com/2959003/140059888-9420d968-31eb-4e93-b39c-7f9ff9e21017.png)

Should we revert it to how it used to be?

Reply:
This should allow the old `TestBackendImport` you removed below to keep working?

Reply:
> Wait, when did this get moved up here? 🤔

It looks like this change was introduced by d8ac13e in #441.

> Should we revert it to how it used to be?

Sure, let's do it! 

> This should allow the old `TestBackendImport` you removed below to keep working?

Actually, I decided to remove `TestBackendImport` because it didn't have a mark (and therefore was never actually run) and only tested a subset of the code path of `TestFrontendImport` anyway.

Reply:
> Actually, I decided to remove TestBackendImport because it didn't have a mark (and therefore was never actually run) and only tested a subset of the code path of TestFrontendImport anyway.

🤦 This has come up before that the SF test suite is set up so that tests _have_ to be marked to be run.

Reply:
The purpose of #536 would be to ensure that this doesn't happen (though it still needs work).

Reply:
Nice! @sduquemesa @antalszava I suddenly realized that #441 is not listed in the changelog, we should definitely add it under 'breaking changes'!

Reply:
Oh, missed that this was moved to the top as well. 👍 for the added comment.

Reply:
Good catch, thank you :) [opened a PR](https://github.com/XanaduAI/strawberryfields/pull/640)